### ISSUE TITLE
Error?

### ISSUE THREAD

**Issue by Zibri** [alxcube/lens#2]
I don't know how to explain it well...
This is the original image:
![image](https://user-images.githubusercontent.com/855176/210248356-56ae7bb3-2859-42ad-873e-90c91f0d62f4.png)

This is the intended result:
![image](https://user-images.githubusercontent.com/855176/210247807-293fc7f1-de57-48b4-b826-9e524456a317.png)
And this is what I get:
![image](https://user-images.githubusercontent.com/855176/210247872-f1f938f0-827f-43c0-88e5-3c3b0aecd6c3.png)

Here is the code:
https://codepen.io/zibri-the-sans/pen/BaPzYBz

```
let result = await lens.distort(image, lens.Distortion.ARC, [angle, rotate], {
    imageVirtualPixelMethod:
      angle === 360
        ? lens.VirtualPixelMethod.HORIZONTAL_TILE
        : lens.VirtualPixelMethod.TRANSPARENT
  });
```

**Comment by Zibri** [comment]
any help?

**Comment by alxcube** [comment]
Hi! it's a bug with average interpolation on transparent images. Add  
`imageInterpolationMethod: lens.InterpolationMethod.INTEGER` to options object to fix that

**Comment by Zibri** [comment]
ohh.. thanks.. it works.
Is there the opposite function?

**Comment by alxcube** [comment]
Fixed bug in v1.0.1

And answering to your second question: no, there's no opposite function in Lens.
